phpcs.xml

```diff
@@ -5,53 +5,35 @@
     <rule ref="Drupal"/>
     <rule ref="DrupalPractice"/>
     <rule ref="Generic.Debug.ESLint"/>
-    <rule ref="PHPCompatibility"/>
+<!--    <rule ref="PHPCompatibility"/>-->
 
-    <arg name="extensions" value="inc,info,install,module,php,profile,test,theme,js,css"/>
+    <arg name="extensions" value="css,inc,info,install,js,module,php,profile,test,theme"/>
     <arg name="colors"/>
     <arg value="sp"/>
     <arg name="parallel" value="75"/>
     <!--Lint code against platform version specified in composer.json
     key "config.platform.php".-->
     <config name="testVersion" value="8.0"/>
 
-    <!-- Exclude civictheme theme assets. -->
-    <exclude-pattern>docroot\/themes\/contrib\/civictheme\/.storybook\/.*</exclude-pattern>
-    <exclude-pattern>docroot\/themes\/contrib\/civictheme\/assets\/.*</exclude-pattern>
-    <exclude-pattern>docroot\/themes\/contrib\/civictheme\/components\/.*</exclude-pattern>
-    <exclude-pattern>docroot\/themes\/contrib\/civictheme\/dist\/.*</exclude-pattern>
-    <exclude-pattern>docroot\/themes\/contrib\/civictheme\/docs\/.*</exclude-pattern>
+    <!-- Exclude node_module directories. -->
```

Comment:
Removed custom exclusions / kept exclusions found in GovCMS ie node_modules etc
[https://github.com/govCMS/GovCMS/blob/2.x-develop/tests/phpcs.govcms.xml](https://github.com/govCMS/GovCMS/blob/2.x-develop/tests/phpcs.govcms.xml)